### PR TITLE
Remove Docker volume mount of "./autogpt:/home/appuser"

### PR DESCRIPTION
**Pull request by Gorgeous-Patrick**
### Background

I could not actually boot up the docker compose. I found out that the folder "autogpt" is mounted as the user home directory. And it is also used as the autogpt package. 

In the dockerfile, the program set up a working home directory and then in the docker compose the system somehow covered the folder.

### Changes

Deleted a docker compose volume.

### Documentation

I think the change is small and it is intuitive to be not documented elsewhere.

### Test Plan

I tried to run it on multiple machines and it works now.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [ x I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by ntindle** [comment]
Can you walk me through how this fixes the issue? Not the most familiar with docker and the team members who are should be sleeping right now

File: docker-compose.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ services:
     env_file:
       - .env
     volumes:
```
Comment (julienledortz) [inline docker-compose.yml:12]:
```suggestion
    volumes:
```

**Comment by Gorgeous-Patrick** [comment]
OK. In the dockerfile, you set up the system perfectly. Note that in the dockerfile, you copy the entire autogpt folder into /home/appuser/autogpt. Then when you run `docker run`, the system enters with the ENTRYPOINT command `python -m autogpt`, which imports the package inside the /home/appuser/autogpt folder. 

However, when you start with docker compose, the system reads the docker-compose.yml and in that file, line 13 means mount the autogpt file outside into the system as /home/appuser, which overwrites the system that we have set up in the dockerfile. And the system runs `python -m autogpt`, it cannot find it.

**Comment by Pwuts** [comment]
Closing as deprecated

**Pull request closed without merging**